tour/1_1_tour.js

```diff
@@ -0,0 +1,56 @@
+import {FeatureTour} from "./feature-tour-wagham.mjs";
+
+export class OneDotOneTour extends FeatureTour {
+
+    constructor() {
+      super({
+        title: "Benvenuto a Wagham!",
+        description: "Panoramica sulla creazione personaggio a Wagham",
+        canBeResumed: true,
+        display: true,
+        version: "1.1.0",

```

Comment:
Partiamo da v1? io partirei da `0.1.0`, visto che non e' ancora in uso

Reply:
no giusto 0.1.0